atat/domain/portfolio_roles.py

```diff
@@ -24,6 +24,22 @@ def get(cls, portfolio_id, user_id):
 
         return portfolio_role
 
+    @classmethod
+    def get_all_by_user(cls, user_id: str = None) -> [PortfolioRole]:
```

Comment:
```suggestion
    def get_all_by_user(cls, user_id: Optional[str] = None) -> List[PortfolioRole]:
```

This is not a valid type annotation as-written. We should go over type hints at some point. The new code will require also importing `from typing import List, Optional` if that's not already imported.

Reply:
Doing the librarires now.